File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        // ArrayList list = new ArrayList();  
        ArrayList<Boolean> list = new ArrayList<Boolean>();
        int max = candies[0];
        int[] candies1=new int[candies.length];
        for(int i=1;i<candies.length;i++){
            if(candies[i]>max){
                max=candies[i];
            }
        }
        
        for(int i=0;i<candies.length;i++){
            candies1[i]=candies[i]+extraCandies;
        }
        for(int i=0;i<candies.length;i++){
            if(candies1[i]>=max){
                list.add(true);
                continue;
            }else{
                list.add(false);
                continue;
            }
            
        }
        return list;
    }
}

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    let candies1=[];
    for(let i=0;i<candies.length;i++){
        let sum=candies[i]+extraCandies;
        for(let j=0;j<candies.length;j++){
            if(sum<candies[j]){
                candies1.push(false)
                break;
            }
        }
        if(candies1[i]!==false){
            candies1.push(true);
        }
        
    }
    return candies1;
    
};